### ISSUE TITLE
Directions to PokéStops and Gyms

### ISSUE THREAD

**Issue by blackhattristar** [istornz/iPokeGo#264]
This would be an enhancement. Is it possible to make the PokéStop and Gym annotations clickable which open with directions in Apple Maps? Just like the Pokémon annotations.

This would be wonderful! :)

**Comment by istornz** [comment]
Yes, I just finished working on the Pogom server part, so I'm going to work on this asap :)
